lib/src/utils/event_localizations.dart

```diff
@@ -240,6 +240,11 @@ abstract class EventLocalizations {
         event.senderFromMemoryOrFallback.calcDisplayname(i18n: i18n)),
     EventTypes.CallCandidates: (event, i18n, body) => i18n.sentCallInformations(
         event.senderFromMemoryOrFallback.calcDisplayname(i18n: i18n)),
+    EventTypes.GroupCallMember: (event, i18n, body) {
+      final memberships = event.content.tryGetList('memberships');
+      if (memberships!.isEmpty) return i18n.groupCallEnded;
```

Comment:
Hm that is actually incorrect, you could have 1 group call device indicate the end of 1 group call but then another device could be in a different call in the same room.

Reply:
Deep checking between membership events could be a bit too advanced for our usecase though. For starters can we just do a callId + deviceId check? we can ignore the application and scope for now because they are mostly unused.

Reply:
I think we need the callId + deviceId check only in timeline. Here, we can just check the memberships and conclude
that some call is active if there's at least one membership that hasn't expired yet (regardless of callId). Else, we can conclude some call has ended.

Reply:
@td-famedly Pls let me know if this sounds good here

Reply:
Yes, but do we know where these localizations will be used? AFAIK we also fallback to these on the timeline. Moreover on every member event you will now get a group call started message while it was just a update to the existing member event.